README.md

```diff
@@ -1,6 +1,15 @@
-# `@ubiquity/bounty-bot` the "UbiquiBot"
+```js
+ _____ _              _   _ _     _             _______       _   
+|_   _| |            | | | | |   (_)           (_) ___ \     | |  
+  | | | |__   ___    | | | | |__  _  __ _ _   _ _| |_/ / ___ | |_ 
+  | | | '_ \ / _ \   | | | | '_ \| |/ _` | | | | | ___ \/ _ \| __|
+  | | | | | |  __/   | |_| | |_) | | (_| | |_| | | |_/ / (_) | |_ 
+  |_| |_| |_|\___|    \___/|_.__/|_|\__, |\__,_|_\____/ \___/ \__|
+                                     | |                        
+                                     |_|                                              
+```
```

Comment:
```suggestion
```

I'm not sure that this will render correctly on the README. It's better to make a real image some day in the future.

Reply:
> I'm not sure that this will render correctly on the README. It's better to make a real image some day in the future.

It renders correctly though, i tested it out and it works when i put it in a code block